components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.enrich.ui/src/main/java/org/wso2/carbon/mediator/enrich/ui/EnrichMediator.java

```diff
@@ -177,13 +179,23 @@ public OMElement serialize(OMElement parent) {
 
             if (sourceInlineXML != null && !sourceInlineXML.equals("")) {
```

Comment:
Shall we rename the variable 'sourceInlineXML' ? This gives the impression that the inline string is an xml, and in line 182 we parse the string with a json parser. The code is not readable IMO.

Reply:
+1 fixed